submissions/oluwabusolamiEluyela/html/index.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html>
+  <head>
+<meta charset "UTF-8">
+  <title>justify.com</title>
+  </head>
+
+  <body>
+    <header>
+      <div> 
+       <h1><a href="#">JUSTIFY</a></h1>
+      </div>
+	 <h2><a href="#">Hot Deals</a>
+         <a href="#">Gift Cards</a>
+         <a href="#">Sell</a><c/enter>
+         <a href="#">Book Rentals</a></h2>
+	<ol>
+         <li><a href="#">New Releases</a></li>
+	 <li><a href="#">General Book Shelf</a></li>
+	 <li><a href="#">Today's Deals</a></li>
+         <li><a href="#">Registration</a></li> 
+	 <li><a href="#">Teachers Corner</a></li>
+	 <li><a href="#">Students Corner</a></li>
+         <li><a href="#">Contact</a></li>
+	 <li><a href="#">Order Offline</a></li>
+	</ol>
+  </header>
+   <main>
+    <div>
+     <li>
+      <div>
+       <h2><a href="">Cambridge Checkpoint Mathematics Coursebook 7</a></h2>
+	<img src="image/cambridge-checkpoint-mathematics-coursebook-7.jpg" alt="cambridge-checkpoint-mathematics-coursebook-7"/>
+         <h2>Price: $20.82</h2>
+          <h4>Customers Ratings: 5.0  (110 Reviews)</h4>
+          </div>
+           <div>
+           <h2>Product description</h2>
+             Written by well-respected authors, the Cambridge Checkpoint Mathematics suite provides a comprehensive structured resource 
+	     which covers the full Cambridge Secondary 1 Mathematics framework in three stages. This brightly illustrated Coursebook for 
+	     Stage 7 offers a comprehensive introduction to all topics covered in the syllabus. Worked examples show students how to tackle 
+	     different problems, and plenty of exercise questions prepare students for the different types of question they will face in their 
+	     exam. Coverage of the Problem Solving framework is integrated throughout the course, with questions relating to the framework 
+	     statements highlighted in the Coursebook and Practice Book.
+            </div>
+          </li>
+          <hr />
+          <li>
```

Comment:
You are using `<li>` tag without `<ul>` or `<ol>` tag. This is wrong. And it isn't used properly in this context